packages/xo-web/src/xo-app/host/tab-stats.js

```diff
@@ -32,6 +38,31 @@ export default class HostStats extends Component {
       if (cancelled) {
         return
       }
+      const statsPifs = stats.stats.pifs
+      const { host, pifs } = this.props
+
+      const deviceName = []
+      forEach(host.PIFs, currentPif => {
+        if (pifs[currentPif]) {
```

Comment:
- Use constant variable for `pifs[currentPif]` 
- Explicit condition
- Regroup conditions